Kernel/include/kernelTypes.h

```diff
@@ -47,7 +47,7 @@ typedef struct {
 /**
  * @brief Represents a semaphore.
  */
-typedef int TSem;
+typedef uint8_t Tsem;
```

Comment:
Following our naming conventions
```suggestion
typedef uint8_t TSem;
```